### ISSUE TITLE
No Logo Shown

### ISSUE THREAD

**Issue by chookity-pokk** [emacs-dashboard/emacs-dashboard#425]
Below is my config for `dashboard` and no matter what I try to set my `dashboard-startup-banner` to, it never seems to appear. When I run `(image-type-available-p 'png)` I get back `t`, and I know the paths are correct. When in org mode I can see images there so I know it's not a system-wide problem. I changed the image a few times and shrank it down to 75% of its size to see if the problem was that the image was too big and that didn't change anything. 

Though sometimes when I put in different images the default banner moves to the right or left a bit, but no images appear. 

```
(require 'dashboard)
(dashboard-setup-startup-hook)

(setq dashboard-week-agenda t)
(setq dashboard-banner-logo-title "I'm using Linux. A library that emacs uses to communicate with Intel hardware.") 
(setq dashboard-startup-banner "/home/hank/Downloads/rsz_emacs-colorful.png")
;(setq dashboard-startup-banner "/home/hank/Pictures/emacs-colorful.png")
(setq dashboard-set-heading-icons t)
(setq dashboard-set-file-icons t)
(setq dashboard-set-navigator t)
(setq dashboard-center-content t)
;; To disable shortcut "jump" indicators for each section, set
(setq dashboard-show-shortcuts nil)
```

I am using Emacs 28.2 on Arch Linux. Installed it via Pacman.

**Comment by jcs090218** [comment]
Did it work if you have `dashboard-startup-banner` set to `'official`? 😕

**Comment by ragreenburg** [comment]
Using `(setq dashboard-startup-banner 'official)` unfortunately didn't change anything.

**Comment by jcs090218** [comment]
> Using (setq dashboard-startup-banner 'official) unfortunately didn't change anything.

Did the official image showed up? Or nothing has shown? 😕

**Comment by ragreenburg** [comment]
Sorry, I should have been more clear. The official image didn't show up when I set that value.

**Comment by jcs090218** [comment]
1. Can you try evaluate the value?

```elisp
(message "%s" (dashboard-choose-banner))
```

This print your banner information.

2. Try to place `(dashboard-setup-startup-hook)` at the very bottom.

I am making guesses here since I am not able to reproduce the issue. 😅

**Comment by ragreenburg** [comment]
1.
I get `(:image /home/hank/Pictures/emacs-colorful.png :text /home/hank/.emacs.d/elpa/dashboard-20221121.1809/banners/1.txt)`

2.
That didn't work either, unfortunately.

**Comment by jcs090218** [comment]
Can you try give us a minimal config to reproduce this bug? Everything seems to work, so I suspect there is something wrong with your configuration!? 😅

**Comment by ragreenburg** [comment]
Here is a link to an org write-up of my config to hopefully make it a bit more readable!

https://gitlab.com/greenbeast/dotfiles/-/blob/main/init.org

**Comment by chookity-pokk** [comment]
Weirdest update here. I updated my init.org and it had an error in it so it didn't go through the whole thing (error was on the last 5 lines though). but when that happened the image showed up! Though when I fixed the error the image was gone again. 

I have no clue what's going on there, as my dashboard config hasn't changed but it only works if my init.org has an error in it...

With error:
![image](https://user-images.githubusercontent.com/24358100/216782259-1b91801f-9fd6-494a-9e6c-ad5cb60e47f7.png)


Without:
![image](https://user-images.githubusercontent.com/24358100/216782316-5022240e-8fa6-420c-a2f1-1a4adf323aff.png)
